galley/pkg/runtime/state.go

```diff
@@ -317,3 +319,12 @@ func (s *State) String() string {
 
 	return b.String()
 }
+
+func monitorEntry(resourceKey resource.VersionedKey, added bool) {
+	namespace, name := resourceKey.FullName.InterpretAsNamespaceAndName()
+	count := 1
+	if !added {
+		count = 0
```

Comment:
I am not sure what the semantics of count here? It looks like add/update is 1, but delete is 0. Doesn't look right.

Reply:
its actually a value.. changed it to make it more consistent